sdk/digitaltwins/Azure.DigitalTwins.Core/src/Customized/Models/InnerError.cs

```diff
@@ -6,7 +6,7 @@
 namespace Azure.DigitalTwins.Core
 {
     [CodeGenModel("InnerError")]
-    internal partial class InnerError
+    public partial class InnerError
```

Comment:
Why did this change to public?

Reply:
Because InnerError in ErrorInformation class defines this as public. I guess another route would be to override the accessibility in ErrorInformation.cs so that InnerError is internal there as well - would that be preferable over this? 

Either way the accessibility somewhere has to change since ErrorInformation is now a public class (per our discussion offline of bulk api using this error model in the response)